hugging_face/hugging_face_pipeline_for_monai.ipynb

```diff
@@ -0,0 +1,584 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    monai_unet.unet.load_state_dict(torch.load(weight_dir))
</pre><p>The same as before.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1946/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>